cmake/external_onnx.cmake

```diff
@@ -31,70 +30,31 @@ set(ONNX_GIT_REPO_URL https://github.com/onnx/onnx.git)
 set(ONNX_GIT_BRANCH rel-${ONNX_VERSION})
 
 add_definitions(-DONNX_BUILD_SHARED_LIBS=ON)
-add_definitions(-DONNX_ML=ON)
 
-ExternalProject_Add(
+if (WIN32)
+    string(REPLACE "/W3" "/W0" CMAKE_ORIGINAL_CXX_FLAGS "${CMAKE_ORIGINAL_CXX_FLAGS}")
+else()
+    string(REPLACE "-Werror" "" CMAKE_CXX_FLAGS "${CMAKE_ORIGINAL_CXX_FLAGS}")
+endif()
```

Comment:
This is what is causing the failure with GCPU. What you are doing here is completely overwriting the CMAKE_CXX_FLAGS for the entire ngraph. Don't do this.
By overwriting CMAKE_CXX_FLAGS for ngraph you are disabling dynamic loading of shared backends. Since this is disabled then the GCPU backend is not looked for as it should be. You need to find a way to disable -Werror without overwriting CMAKE_CXX_FLAGS. This file is add to the cmake file with `include` so you are running at the scope of the main cmake file.

Reply:
I removed this code